### ISSUE TITLE
Add identity Object config

### ISSUE THREAD

**Issue by himynamesdave** [signalscorps/file2stix#12]
In STIX Identity Objects define who created an Object. We should allow user to set a config file where they can define

* Organisation Name (required)
* Organisation Description (optional)
* Organisation Class (optional)
  * Selected from [Identity Class Vocabulary](https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_be1dktvcmyu)
* Organisation Sector (optional)
  * Selected from [Industry Sector Vocabulary](https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_oogrswk3onck)

```json
{
  "type": "identity",
  "spec_version": "2.1",
  "id": "identity--<GENERATED BY STIX2 LIBRARY>",
  "created": "<ORGANISATION CREATE DATE>",
  "modified": "<ORGANISATION LAST MODIFIED DATE>",
  "name": "<ORGANISATION NAME>",
  "description": "<ORGANISATION DESCRIPTION>",
  "identity_class": "<ORGANISATION CLASS>",
  "sectors": "<ORGANISATION SECTOR>"
}
```

This identity object(s) do not change (have same ID) once created.

If user changes identity values in config, they should be able to run a regeneration script to generate (and overwrite) old object

These identity Objects are then used inside of all Objects created from TLP:AMBER reports (including relationships -- see #11 )

```json
  {
    "type": "indicator",
    "spec_version": "2.1",
    "id": "indicator--<GENERATED BY STIX2 LIBRARY>",
    "created_by_ref": "identity--<IDENTITY ID>",
    "created": "<FIRST EXTRACTED DATETIME>",
    "modified": "<LAST EXTRACTED DATETIME>",
    "indicator_types": ["malicious-activity"],
    "name": "ipv4: <EXTRACTED IPV4 OBSERVABLE VALUE>",
    "pattern_type": "stix",
    "pattern": "[ ipv4-addr:value = '<EXTRACTED IPV4 OBSERVABLE VALUE>' ]",
    "valid_from": "<FIRST EXTRACTED DATETIME>",
    "object_marking_refs": [
      "marking-definition--<TLP LEVEL SET>"
    ],
    "defanged": true,
    "x_warning_list_match": ["<NAME OF WARNING LIST MATCHED>"]
  }
```

**Comment by himynamesdave** [comment]
@shanthanu9 not quite.

All old STIX Objects do not change. `created_by_ref` is static.

In 

> If user changes identity values in config, they should be able to run a regeneration script to generate (and overwrite) old object

Ignore the word overwrite. It is not correct @shanthanu9 

"object" in this scenario is the identity object.

If user runs "update identity" (or whatever), the identity object is updated. The identity object used from t+0 should represent the new identity not the old one for all now objects.

**Comment by himynamesdave** [comment]
@shanthanu9 not quite, the identity objects need to exist in the stix bundles

**Comment by himynamesdave** [comment]
i still don't see this in the STIX Bundle

```
{
    "type": "bundle",
    "id": "bundle--e64cf399-4365-4638-87e3-f4cbe2484d14",
    "objects": [
        {
            "type": "extension-definition",
            "spec_version": "2.1",
            "id": "extension-definition--c8ea5ecb-f4a3-45e7-94de-9b9ba05161af",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-01-01T00:00:00.000Z",
            "modified": "2022-01-01T00:00:00.000Z",
            "name": "MISP Warning Lists",
            "description": "This schema adds MISP Warning List matches to extracted Objects",
            "schema": "https://github.com/MISP/misp-warninglists",
            "version": "1.0",
            "extension_types": [
                "property-extension"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--f1dac064-9e70-45f9-859c-32ac00265596",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:43:59.180901Z",
            "modified": "2022-09-06T05:45:01.784042Z",
            "name": "ipv4: 198.51.100.5",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.51.100.5' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:43:59.180901Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--2431ef23-2b1c-451b-85f3-52d4092ca6db",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:43:59.683935Z",
            "modified": "2022-09-06T05:45:02.306015Z",
            "name": "ipv4: 198.0.103.12",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:43:59.683935Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--1554a3e7-a961-4c12-99d3-3dad9dc3e8e9",
            "created": "2022-09-06T05:34:57.207787Z",
            "modified": "2022-09-06T05:45:02.830603Z",
            "name": "ipv4: 198.0.103.12/24",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12/24' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:34:57.207787Z",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--8034f499-67f7-471a-93e5-35b4d47e7693",
            "created": "2022-09-06T05:34:57.295944Z",
            "modified": "2022-09-06T05:45:03.955023Z",
            "name": "ipv4: 198.0.103.12:8000",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' AND network-traffic:dst_port = '8000' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:34:57.295944Z",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--602be8de-0f81-4ea5-b559-41bf40b7b4f8",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:44:00.636477Z",
            "modified": "2022-09-06T05:45:03.279862Z",
            "name": "ipv4: 198.0.103.12:8",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' AND network-traffic:dst_port = '8' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:44:00.636477Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--6ab7cbac-0390-4557-b119-80c5f54c9dd8",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:44:00.840762Z",
            "modified": "2022-09-06T05:45:03.509134Z",
            "name": "ipv4: 198.0.103.12:80",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' AND network-traffic:dst_port = '80' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:44:00.840762Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--9a66a479-d56a-49c5-82ea-6b11d2c23f8d",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:44:01.110465Z",
            "modified": "2022-09-06T05:45:03.733033Z",
            "name": "ipv4: 198.0.103.12:800",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' AND network-traffic:dst_port = '800' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:44:01.110465Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--b9d44674-c380-4b42-8fc4-c9ce3dca498b",
            "created_by_ref": "identity--acf55024-6bbe-486f-a27a-7967559324f4",
            "created": "2022-09-06T05:44:01.521134Z",
            "modified": "2022-09-06T05:45:04.193052Z",
            "name": "ipv4: 198.0.103.12:65535",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ ipv4-addr:value = '198.0.103.12' AND network-traffic:dst_port = '65535' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-09-06T05:44:01.521134Z",
            "object_marking_refs": [
                "marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da"
            ]
        },
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "created": "2022-09-06T05:45:17.178781Z",
            "modified": "2022-09-06T05:45:17.178781Z",
            "name": "File converted: ipv4.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-09-06T07:45:17.178769Z",
            "object_refs": [
                "indicator--f1dac064-9e70-45f9-859c-32ac00265596",
                "indicator--2431ef23-2b1c-451b-85f3-52d4092ca6db",
                "indicator--1554a3e7-a961-4c12-99d3-3dad9dc3e8e9",
                "indicator--8034f499-67f7-471a-93e5-35b4d47e7693",
                "indicator--602be8de-0f81-4ea5-b559-41bf40b7b4f8",
                "indicator--6ab7cbac-0390-4557-b119-80c5f54c9dd8",
                "indicator--9a66a479-d56a-49c5-82ea-6b11d2c23f8d",
                "indicator--b9d44674-c380-4b42-8fc4-c9ce3dca498b"
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--2953d4dd-884e-4ce2-9e45-0e34bf86cb21",
            "created": "2022-09-06T05:45:17.179141Z",
            "modified": "2022-09-06T05:45:17.179141Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--f1dac064-9e70-45f9-859c-32ac00265596",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--a9270548-6574-4205-b2ae-823b9f08e60d",
            "created": "2022-09-06T05:45:17.17934Z",
            "modified": "2022-09-06T05:45:17.17934Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--2431ef23-2b1c-451b-85f3-52d4092ca6db",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--d95ff19a-81e5-4f4b-b56b-0dcd0eb14a2c",
            "created": "2022-09-06T05:45:17.179523Z",
            "modified": "2022-09-06T05:45:17.179523Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--1554a3e7-a961-4c12-99d3-3dad9dc3e8e9",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--b5aeae07-7ab5-4ac5-9922-a7d3a7566be6",
            "created": "2022-09-06T05:45:17.17971Z",
            "modified": "2022-09-06T05:45:17.17971Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--8034f499-67f7-471a-93e5-35b4d47e7693",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--bc8ef1d3-3832-4683-8828-33f5d1c1417d",
            "created": "2022-09-06T05:45:17.179896Z",
            "modified": "2022-09-06T05:45:17.179896Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--602be8de-0f81-4ea5-b559-41bf40b7b4f8",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--2f80b302-7c84-4970-b54d-a0d03450d486",
            "created": "2022-09-06T05:45:17.180076Z",
            "modified": "2022-09-06T05:45:17.180076Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--6ab7cbac-0390-4557-b119-80c5f54c9dd8",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--b2c89646-9275-4bd4-9d23-3df8160b0a8a",
            "created": "2022-09-06T05:45:17.180256Z",
            "modified": "2022-09-06T05:45:17.180256Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--9a66a479-d56a-49c5-82ea-6b11d2c23f8d",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--1a814f83-ad60-43e8-adf7-05b95b35ebc0",
            "created": "2022-09-06T05:45:17.180433Z",
            "modified": "2022-09-06T05:45:17.180433Z",
            "relationship_type": "default-extract",
            "source_ref": "report--6a2615b8-d32c-4855-ad2f-db3ae069c223",
            "target_ref": "indicator--b9d44674-c380-4b42-8fc4-c9ce3dca498b",
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ]
        }
    ]
}
```
